c/tskit/tables.c

```diff
@@ -7566,9 +7686,10 @@ tsk_table_collection_deduplicate_sites(
         goto out;
     }
     /* Check everything except site duplicates (which we expect) and
-     * edge indexes (which we don't use) */
+     * edge indexes and mutation timesc (which we don't use) */
```

Comment:
"timesc"

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/672/commits/79fb3a06a109d1b5e37812859c26d33357d38a82